libs/core/langchain_core/messages/tool.py

```diff
@@ -83,6 +109,7 @@ def __add__(self, other: Any) -> BaseMessageChunk:  # type: ignore
             return self.__class__(
                 tool_call_id=self.tool_call_id,
                 content=merge_content(self.content, other.content),
+                raw_output=merge_dicts(self.raw_output, other.raw_output),
```

Comment:
I don't think this is a concern because I'm not sure how often we're summing ToolCallChunks, but this might typically error, e.g.,
```python
message_1 = ToolMessageChunk("foo", raw_output={"bar": {"baz": 123}}, tool_call_id="1")
message_2 = ToolMessageChunk("foo", raw_output={"bar": {"baz": 234}}, tool_call_id="1")
message_1 + message_2
```
```
TypeError: Additional kwargs key baz already exists in left dict and value has unsupported type <class 'int'>
```

Reply:
yea, but my thinking is that's an inavlid stream of chunks then. not clear how to combine them